tests/test_simplelog.py

```diff
@@ -0,0 +1,192 @@
+import sys
+
+import re
+import click
+import logging
+
+log_stream = None
+
+
+def tool_resetlog():
```

Comment:
Click also captures stdout and stderr and stores it in result.output. Sorry for the earlier comment, bow Idon't think pytest's capturing methods will work in this case.